src/components/tree-views-container.jsx

```diff
@@ -130,6 +129,7 @@ function updatedTree(prevRoots, prevMap, nodeId, children) {
         ...Array.prototype.concat(
             ...prevUuids
                 .filter((u) => !nextUuids.has(u))
+                .filter((uuid) => !(nodeId && prevMap[uuid] && prevMap[uuid].parentUuid !== nodeId)) // if the element is moved but not deleted
```

Comment:
I don't think we should cange this really sensible code indeed.
What's the problem here ?